components/post_view/commented_on/commented_on.jsx

```diff
@@ -61,12 +65,16 @@ export default class CommentedOn extends PureComponent {
         );
 
         let message = '';
-        if (post.message) {
-            message = Utils.replaceHtmlEntities(post.message);
+        if (post.message === '' && post.props && post.props.attachments) {
```

Comment:
I think the priority for each case is mixed up here. Files are supposed to be shown before message attachments, and this is making it so that a post with file attachments doesn't show the message. It should be something like
```
let message = '';
if (post.message) {
    message = post.message;
} else if (post.file_ids && post.file_ids.length > 0) {
    message = <CommentedOnFilesMessage .../>;
} else if (post.props && post.props.attachments && post.props.attachments.length > 0) {
    message = ...;
}
```

Reply:
Should files be shown before attachments?

If a webhook has attachments and files in the message I thought we should stick to attachments pretext.

Reply:
Fixed this. Thanks